### PR TITLE
schek/add_test_delete_bookmark_from_other_bookmarks

### PR DESCRIPTION
**Pull request by alexandruschek**
### Description

Created a new test for verifying that a user can Delete a bookmark from 'Other Bookmarks' folder 

### Bugzilla bug ID
1910731
**Testrail:**https://mozilla.testrail.io/index.php?/cases/view/2084524
**Link:**https://bugzilla.mozilla.org/show_bug.cgi?id=1910731

### Type of change

Please delete options that are not relevant.

- [x] New Test

### How does this resolve / make progress on that bug?
Completed

### Screenshots / Explanations

N/A
###Comments / Concerns

N/A

### CODE REVIEW

File: tests/bookmarks_and_history/test_delete_other_bookmars.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+BOOKMARK_URL_2 = "about:cache"
+
+
+WIN_GHA = environ.get("GITHUB_ACTIONS") == "true" and sys.platform.startswith("win")
+
+
+@pytest.mark.skipif(WIN_GHA, reason="Test unstable in Windows Github Actions")
+def test_delete_other_bookmarks(driver: Firefox):
+    """
+        C2084524: Verify that a user can Delete a bookmark from 'Other Bookmarks' folder
```
Comment (ben-c-at-moz) [inline tests/bookmarks_and_history/test_delete_other_bookmars.py:23]: These days we need to indicate the test case in a fixture. Outside of this test function, please write a fixture function:

```python
@pytest.fixture()
def test_case():
    return "2084524"
```

File: tests/bookmarks_and_history/test_delete_other_bookmars.py
Comment (ben-c-at-moz) [inline tests/bookmarks_and_history/test_delete_other_bookmars.py:1]: Typo in filename, should end in `bookmarks.py`.

**Review by ben-c-at-moz — CHANGES_REQUESTED** [review]
Test looks and works great, just a couple of superficial things to fix, and we'll be good to merge.

File: tests/bookmarks_and_history/test_delete_other_bookmars.py
Comment (alexandruschek) [reply tests/bookmarks_and_history/test_delete_other_bookmars.py:1]: Thank you, I renamed the file and added a "k".

File: tests/bookmarks_and_history/test_delete_other_bookmars.py (hunk 1 above)
Comment (alexandruschek) [reply tests/bookmarks_and_history/test_delete_other_bookmars.py]: Fixture added

**Review by ben-c-at-moz — APPROVED** [review]
Thanks for taking care of that! Ready to merge

**Pull request merged**